### PR TITLE
[CONSUL-242] Create Jaeger Windows Image

### PR DESCRIPTION
**Pull request by joselo85**
### Description
Created Dockerfile to build Jaeger-Windows image. Replicating the all-in-one image available for Linux.

### CODE REVIEW

File: build-support-windows/build-images.sh [hunk 1]
```diff
@@ -32,4 +32,7 @@ docker build -t "${HASHICORP_DOCKER_PROXY}/windows/fortio" -f Dockerfile-fortio-
 # Build Windows Socat Image
 docker build -t "${HASHICORP_DOCKER_PROXY}/windows/socat" -f Dockerfile-socat-windows .
 
+# Buiild Windows Jaeger Image
+docker build -t "${HASHICORP_DOCKER_PROXY}/windows/socat" -f Dockerfile-jaeger-windows .
```
Comment (ezfepo) [inline build-support-windows/build-images.sh]: wrong tag

File: test/integration/connect/envoy/run-tests.windows.sh [hunk 2]
```diff
@@ -821,7 +821,7 @@ function run_container_jaeger {
   docker.exe run -d --name $(container_name) \
     $WORKDIR_SNIPPET \
     $(network_snippet primary) \
-    "${HASHICORP_DOCKER_PROXY}/jaegertracing/all-in-one:1.11" \
+    "${HASHICORP_DOCKER_PROXY}/windows/jaegertracing/all-in-one:1.11" \
```
Comment (ezfepo) [inline test/integration/connect/envoy/run-tests.windows.sh]:
```suggestion
    "${HASHICORP_DOCKER_PROXY}/windows/jaegertracing" \
```

File: build-support-windows/build-images.sh [hunk 3]
```diff
@@ -32,4 +32,7 @@ docker build -t "${HASHICORP_DOCKER_PROXY}/windows/fortio" -f Dockerfile-fortio-
 # Build Windows Socat Image
 docker build -t "${HASHICORP_DOCKER_PROXY}/windows/socat" -f Dockerfile-socat-windows .
 
+# Buiild Windows Jaeger Image
```
Comment (ezfepo) [inline build-support-windows/build-images.sh]:
```suggestion
# Buiild Windows Jaegertracing Image
```

also I prefer to have the images sorted by name...

**Review by ezfepo — CHANGES_REQUESTED** [review]

File: build-support-windows/docker.windows.md [hunk 4]
```diff
@@ -8,6 +8,7 @@
 - [Dockerfile-consul-dev-windows](#dockerfile-consul-dev-windows)
 - [Dockerfile-fortio-windows](#dockerfile-fortio-windows)
 - [Dockerfile-socat-windows](#dockerfile-socat-windows)
+- [Dockerfile-jaeger-windows](#dockerfile-jaeger-windows)
```
Comment (ezfepo) [inline build-support-windows/docker.windows.md]: let's place whole jaeger documentation, before socat. So it would be sorted by name

**Review by ezfepo — CHANGES_REQUESTED** [review]

File: build-support-windows/build-images.sh [hunk 5]
```diff
@@ -29,6 +29,9 @@ docker build -t "${HASHICORP_DOCKER_PROXY}/windows/bats:1.7.0" -f Dockerfile-bat
 # Build Windows Fortio Image
 docker build -t "${HASHICORP_DOCKER_PROXY}/windows/fortio" -f Dockerfile-fortio-windows .
 
+# Build Windows Jaegertracing Image
+docker build -t "${HASHICORP_DOCKER_PROXY}/windows/jaegertracing" -f Dockerfile-jaeger-windows .
```
Comment (ezfepo) [inline build-support-windows/build-images.sh]:
```suggestion
docker build -t "${HASHICORP_DOCKER_PROXY}/windows/jaegertracing" -f Dockerfile-jaegertracing-windows .
```

File: build-support-windows/docker.windows.md (hunk 4 above)
Comment (ezfepo) [reply build-support-windows/docker.windows.md]:
```suggestion
- [Dockerfile-jaegertracing-windows](#dockerfile-jaegertracing-windows)
```

File: build-support-windows/docker.windows.md [hunk 6]
```diff
@@ -98,6 +100,35 @@ docker run --rm -p 8080:8080 --name fortio fortio
 
 If everything works properly you should openning the browser and check that the Fortio server running on: `http://localhost:8080/fortio`
 
+## Dockerfile-jaeger-windows
```
Comment (ezfepo) [inline build-support-windows/docker.windows.md]:
```suggestion
## Dockerfile-jaegertracing-windows
```

File: build-support-windows/docker.windows.md [hunk 7]
```diff
@@ -98,6 +100,35 @@ docker run --rm -p 8080:8080 --name fortio fortio
...
+
+The all-in-one image was replaced by a [windows/servercore image](https://hub.docker.com/_/microsoft-windows-servercore) image, where we download the official jaegertracing binaries for Windows. Then we replicate everything that is being done in the all-in-one Docker image available for Linux.
+To build this image you need to run the following command on your terminal:
+
+```shell
+docker build -t jaeger -f Dockerfile-jaeger-windows .
```
Comment (ezfepo) [inline build-support-windows/docker.windows.md]:
```suggestion
docker build -t jaegertracing -f Dockerfile-jaegertracing-windows .
```

File: build-support-windows/docker.windows.md [hunk 8]
```diff
@@ -98,6 +100,35 @@ docker run --rm -p 8080:8080 --name fortio fortio
...
+To build the images, it is necessary to open a Git bash terminal and run
+
+```shell
+./build-images.sh
+```
+
+You can test the built file by running the following command:
+
+```shell
+docker run --rm --name jaeger version
```
Comment (ezfepo) [inline build-support-windows/docker.windows.md]:
```suggestion
docker run --rm --name jaegertracing version
```

File: build-support-windows/docker.windows.md [hunk 9]
```diff
@@ -98,6 +100,35 @@ docker run --rm -p 8080:8080 --name fortio fortio
...
+
+## Build images
+
+To build the images, it is necessary to open a Git bash terminal and run
+
+```shell
+./build-images.sh
+```
+
+You can test the built file by running the following command:
```
Comment (ezfepo) [inline build-support-windows/docker.windows.md:112]: this section belongs to jaegertracing 🤣

**Review by ezfepo — CHANGES_REQUESTED** [review]

File: build-support-windows/docker.windows.md (hunk 9 above)
Comment (joselo85) [reply build-support-windows/docker.windows.md:112]: Third time is the charm!

**Review by ezfepo — APPROVED** [review]

**Pull request merged**